NEWS

```diff
@@ -12,6 +12,10 @@
   which have empty repository scopes (gh#SUSE/machinery#1268)
 * Inspection of unmanaged-files is now also using the faster machinery-helper
   when the files are extracted
+* Introduce format version 5 which adds the "environment" scope
```

Comment:
Nearly all other format changes look like this:
`Introduce system description format version 3 (see https://github.com/SUSE/machinery/wiki/System-Description-Format#version-3)`